### PR TITLE
Third Week

### PR DESCRIPTION
**Pull request by aces47**
Add controllers and view for VehicleType and Detail entities

### CODE REVIEW

File: WebAutopark/Startup.cs [hunk 1]
```diff
@@ -48,6 +49,10 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
                 app.UseHsts();
             }
+            app.UseRequestLocalization();
+            var cultureInfo = new CultureInfo("en-US");
+            CultureInfo.DefaultThreadCurrentCulture = cultureInfo;
+            CultureInfo.DefaultThreadCurrentUICulture = cultureInfo;
```
Comment (PushMeNow) [inline WebAutopark/Startup.cs:55]: UseRequestLocalization has additional options for culture describing. Move culture initialization there.

File: WebAutopark/Views/Detail/Delete.cshtml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+<div>
+    <h4>Detail</h4>
+    <hr />
+    <dl class="row">
+        <dt class = "col-sm-2" hidden>
+            @Html.DisplayNameFor(model => model.DetailId)
+        </dt>
+        <dd class = "col-sm-10" hidden>
+            @Html.HiddenFor(model => model.DetailId)
+        </dd>
```
[the remark this answers is not in the document]
Comment (aces47) [reply WebAutopark/Views/Detail/Delete.cshtml:19]: Fixed

File: WebAutopark/Views/Detail/Index.cshtml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+<p>
+    <a asp-action="Create">Create New</a>
+</p>
+<table class="table">
+    <thead>
+        <tr>
+            <th hidden>
+                @Html.DisplayNameFor(model => model.DetailId)
+            </th>
```
[the remark this answers is not in the document]
Comment (aces47) [reply WebAutopark/Views/Detail/Index.cshtml:17]: Fixed

File: WebAutopark/Views/Detail/Update.cshtml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+<h4>Detail</h4>
+<hr />
+<div class="row">
+    <div class="col-md-4">
+        <form asp-action="Update">
+            <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+            <div class="form-group" hidden>
+                <label asp-for="DetailId" class="control-label"></label>
+                <input asp-for="DetailId" class="form-control" />
+                <span asp-validation-for="DetailId" class="text-danger"></span>
```
[the remark this answers is not in the document]
Comment (aces47) [reply WebAutopark/Views/Detail/Update.cshtml:18]: Fixed

File: WebAutopark/Views/Detail/Update.cshtml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+<h1>Update</h1>
+
+<h4>Detail</h4>
+<hr />
+<div class="row">
+    <div class="col-md-4">
+        <form asp-action="Update">
+            <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+            <div class="form-group" hidden>
+                <label asp-for="DetailId" class="control-label"></label>
```
[the remark this answers is not in the document]
Comment (aces47) [reply WebAutopark/Views/Detail/Update.cshtml:16]: Fixed

**Review by kuzmiich — APPROVED** [review]
Good job

**Pull request merged**